controllers/keysend.ctrl.go

```diff
@@ -0,0 +1,98 @@
+package controllers
```

Comment:
@kiwiidb compared to your changes, I renamed this file to have ctrl suffix and followed your todo hints to implement endpoint